src/main.ts

```diff
@@ -135,22 +136,43 @@ export async function run(): Promise<void> {
     logInfo(`Running: ${commandString}`);
 
     // Get output of gcloud cmd.
-    const output = await getExecOutput(toolCommand, appDeployCmd, options);
-    if (output.exitCode !== 0) {
-      const errMsg = output.stderr || `command exited ${output.exitCode}, but stderr had no output`;
+    const deployOutput = await getExecOutput(toolCommand, appDeployCmd, options);
+    if (deployOutput.exitCode !== 0) {
+      const errMsg =
+        deployOutput.stderr || `command exited ${deployOutput.exitCode}, but stderr had no output`;
       throw new Error(`failed to execute gcloud command \`${commandString}\`: ${errMsg}`);
     }
 
-    // Set url as output.
-    const response = parseDeployResponse(output.stdout);
-    if (response) {
-      setOutput('name', response.name);
-      setOutput('serviceAccountEmail', response.serviceAccountEmail);
-      setOutput('versionURL', response.versionURL);
```

Comment:
These two are technically breaking changes, but they were never published in the action.yml or documentation. I'm happy to add them for BC if we think it will be an issue.